Sources/DistributedActors/Cluster/Leadership.swift

```diff
@@ -112,7 +112,9 @@ extension Leadership {
         var behavior: _Behavior<Cluster.Event> {
             .setup { context in
                 context.log.trace("Configured with \(self.election)")
-                context.system.cluster.events.subscribe(context.myself)
+                Task {
+                    await context.system.cluster.events.subscribe(context.myself)
```

Comment:
We can't touch behavior actor context from a different task ,this is racy -- needs the same fix as the other place, get the events into a `let` before the task

Reply:
Eh sorry I merged before you had a chance to address this one @yim-lee